tests/_support/Database/Migrations/20160428212500_Create_test_tables.php

```diff
@@ -47,16 +47,22 @@ public function up(): void
 
         // Database Type test table
         // missing types:
-        //     TINYINT,MEDIUMINT,BIT,YEAR,BINARY,VARBINARY,TINYTEXT,LONGTEXT,
-        //     JSON,Spatial data types
+        //   TINYINT,MEDIUMINT,BIT,YEAR,BINARY,VARBINARY,TINYTEXT,LONGTEXT,
+        //   JSON,Spatial data types
         // `id` must be INTEGER else SQLite3 error on not null for autoincrement field.
         $data_type_fields = [
-            'id'              => ['type' => 'INTEGER', 'constraint' => 20, 'auto_increment' => true],
-            'type_varchar'    => ['type' => 'VARCHAR', 'constraint' => 40, 'null' => true],
-            'type_char'       => ['type' => 'CHAR', 'constraint' => 10, 'null' => true],
-            'type_text'       => ['type' => 'TEXT', 'null' => true],
-            'type_smallint'   => ['type' => 'SMALLINT', 'null' => true],
-            'type_integer'    => ['type' => 'INTEGER', 'null' => true],
+            'id'           => ['type' => 'INTEGER', 'constraint' => 20, 'auto_increment' => true],
+            'type_varchar' => ['type' => 'VARCHAR', 'constraint' => 40, 'null' => true],
+            'type_char'    => ['type' => 'CHAR', 'constraint' => 10, 'null' => true],
+            // TEXT should not be used on SQLSRV. It is deprecated.
+            'type_text'     => ['type' => 'TEXT', 'null' => true],
+            'type_smallint' => ['type' => 'SMALLINT', 'null' => true],
+            'type_integer'  => ['type' => 'INTEGER', 'null' => true],
+            // FLOAT should not be used on MySQL.
+            //   CREATE TABLE t (f FLOAT, d DOUBLE);
+            //   INSERT INTO t VALUES(99.9, 99.9);
+            //   SELECT * FROM t WHERE f=99.9; // Empty set
```

Comment:
Float is an approximation of a number. The following would work:
```SQL
SELECT * FROM t WHERE f > 99.89 AND f < 99.91;
```

Reply:
Yes, and we cannot use `seeInDatabase()`.